msg/TecsStatus.msg

```diff
@@ -33,3 +33,4 @@ uint8 TECS_MODE_LAND = 3
 uint8 TECS_MODE_LAND_THROTTLELIM = 4
 uint8 TECS_MODE_BAD_DESCENT = 5
 uint8 TECS_MODE_CLIMBOUT = 6
+uint8 TECS_MODE_STALL = 7
```

Comment:
```suggestion
uint8 TECS_MODE_STALL_PREVENTION = 7
```

Reply:
I thought it would make it a bit more clear what we are trying to do, otherwise it sounds a bit like we are supporting a mode where we fly around stalled :)